action.yml

```diff
@@ -7,12 +7,18 @@ inputs:
   service_account:
     description: 'Service account for authentication. It could be file content or file itself.'
     required: true
+  platform:
+    description: 'Platform to run [ios | android]'
+    required: true
+  flank_configuration_file:
+    description: 'Flank configuration file'
+    required: true
 runs:
   using: "composite"
   steps:
     - name: Download flankScripts
       run: |
-        curl -L https://github.com/Flank/flank/releases/download/flank-scripts-1.9.0/flank-scripts.jar --output flank-scripts.jar
+        curl -L https://github.com/Flank/flank/releases/download/flank-scripts-1.9.1/flank-scripts.jar --output flank-scripts.jar
```

Comment:
is there a possibility to always retrieve the latest?
Therefore we dont need to constantly change the version number?

Reply:
@Sloox 
It is by design because we are sure that we will use a stable version tested with this Github action